src/InputContainer.test.jsx

```diff
@@ -0,0 +1,52 @@
+import React from 'react';
+import { render, fireEvent } from '@testing-library/react';
+import { useSelector, useDispatch } from 'react-redux';
+
+import InputContainer from './InputContainer';
+import { ADD_TASK, UPDATE_TASK } from './actions';
+
+jest.mock('react-redux');
+
+describe('InputContainer', () => {
+  it('should render label, input, button', () => {
```

Comment:
```suggestion
  it('should renders label, input, button', () => {
```

여기서 it(InputContainer)은 3인칭 단수이기 때문에 s를 붙여주는게 더 자연스럽지 않을까요? 🤔